demo/compose.yaml

```
@@ -0,0 +1,24 @@
+services:
+        
+  zed:
+    image: ${ZED_IMAGE}
+    container_name: zed
+    network_mode: host
+    runtime: nvidia
+    tty: true
+    volumes:
+      - ./params:/ros_ws/src/zed-ros-wrapper/zed_wrapper/params
+      - /dev:/dev 
+      - /tmp/zed_ai/:/usr/local/zed/resources/
+    privileged: true
+    command: ros2 launch zed_wrapper zed2.launch.py
```

```suggestion
    command: ros2 launch zed_wrapper ${CAMERA_MODEL}.launch.py
```
I guess launch file should be env